core/opendaq/reader/src/multi_reader_impl.cpp

```diff
@@ -311,7 +312,7 @@ ListPtr<IInputPortConfig> MultiReaderImpl::CheckPreconditions(const ListPtr<ICom
                 throw InvalidParameterException("Cannot pass both input ports and signals as items");
 
             if (overrideMethod && port.getConnection().assigned())
-                throw InvalidParameterException("Signal has to be connected to port after reader is created");
+                throw InvalidParameterException("Signal has to be connected to the port before the reader is created");
```

Comment:
Isn't this the opposite? The signal must not yet be connected before a the reader is created.